frappe/tests/test_background_jobs.py

```diff
@@ -44,6 +52,79 @@ def test_enqueue_at_front(self):
 		# lesser is earlier
 		self.assertTrue(high_priority_job.get_position() < low_priority_job.get_position())
 
+	def test_enqueue_call(self):
+		with patch.object(Queue, "enqueue_call") as mock_enqueue_call:
+			frappe.enqueue(
+				"frappe.handler.ping",
+				queue="short",
+				timeout=300,
+				kwargs={"site": frappe.local.site},
+			)
+
+			mock_enqueue_call.assert_called_once_with(
+				execute_job,
+				on_success=None,
+				on_failure=None,
```

Comment:
This test is failing, mostly because on_success and on_failure don't exist on stable branches. 

(we need to remove them entirely tbh)